perun-openapi/openapi.yml

```diff
@@ -7372,6 +7372,20 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /urlinjsonout/usersManager/getUsersBySpecificUser:
+    post:
+      tags:
+        - UsersManager
+      operationId: getUsersBySpecificUser
+      summary: Return all users who owns the specific user.
+      parameters:
+        - $ref: '#/components/parameters/specificUserId'
```

Comment:
I think the correct parameter should be '#/components/parameters/specificUser'
You can find the parameter name in UsersManagerMethod.java in the call of the getUsersBySpecificUser method.

Reply:
This is actually correct, specificUserId is defined with the name specificUser, it is the same as with userId and user.

Reply:
Sorry, you are right. Missed the reference.